manifests/logging-stack/output-elasticsearch.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     logging.kubesphere.io/enabled: "true"
     logging.kubesphere.io/component: logging
 spec:
-  match: kube.*
+  matchRegex: (?:kube|service)\.(.*)
```

Comment:
I think all the 3 outputs should make similar changes:
![image](https://user-images.githubusercontent.com/18525465/103070847-5b8de380-45fd-11eb-9dd9-0df31754ac3a.png)